apps/qubit/modules/repository/templates/_browseTableView.php

```diff
@@ -42,6 +42,7 @@
       </th>
 
       <th style="width 3%">
+        <span class="sr-only"><?php echo __('Clipboard'); ?></span>
```

Comment:
I didn't know this was available in Bootstrap 2!

Reply:
Apparently it's also part of Font Awesome.